styles/homemaincontent.css

```diff
@@ -0,0 +1,244 @@
+body {
+    min-height: 100%;
+    font-family: "Poppins", sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+  
+.container {
+    display: flex;
+    flex-grow: 1;
+}
+  
+.sidebar {
+    flex-basis: 25%;
+}
+  
+.main-content {
+    background-image: url("../images/pexels-c-reiners-907786.jpg") ;
+    background-repeat: no-repeat;
+    background-size: cover;
+    flex-grow: 1;
+    color: white;
+    animation: background-change 25s linear infinite;
+}
+  
+.content-title {
+    font-size: 2rem;
+    text-align: center;
+    text-transform: uppercase;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+  
+.main-content-details {
+    margin: 8px;
+    padding: 0 8px;
+}
+  
+.main-content-details h5 {
+    margin: 8px;
+    padding: 8px;
+    font-size: 18px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.main-content-details p {
+    margin: 8px;
+    padding: 8px;
+    font-size: 14px;
+}
+
+.sub-headings {
+    background-color: #0f403d;
+    clip-path: polygon(100% 0%, 75% 50%, 100% 100%, 25% 100%, 0% 50%, 25% 0%);
+    color: white;
+    margin: 16px;
+    padding: 16px;
+    font-size: 24px;
+    text-align: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+  
+.facilities {
+    display: grid;
+    grid-template: 500px / repeat(3 , 1fr);
+    gap: 16px;
+    border: 10px solid #1a9973;
+}
+  
+.facilities-grid {
+    border: 1px solid #0f403d;
+    box-shadow: 2px 2px 2px;
+}
+  
+.facilities-grid img {
+    width: 100%;
+    height: auto;
+}
+  
+.facilities-grid h4 {
+    margin: 8px;
+    padding: 8px;
+    font-size: 16px;
+}
+  
+.facilities-grid p { 
+    margin: 8px;
+    padding: 8px;
+    font-size: 14px;
+}
+  
+.highlight {
+    cursor: pointer; 
+    transform: scale(1.05);
```

Comment:
The .highlight class adds a scale transform. You might add a smooth transition for better visual